test/daoreg.offers.test.js

```diff
@@ -574,12 +551,14 @@ describe('Tests for offers in dao registry', async function () {
     })
     console.log('BEFORE offer_sell balancetable  is: ', JSON.stringify(balancetable, null , ' '), '\n')
 
-    
-    const offer_sell = await OffersFactory.createWithDefaults({ creator: bob, type: OfferConstants.sell })
+    // Act
+    const offer_sell = await OffersFactory.createWithDefaults({ creator: bob, type: OfferConstants.sell, quantity:newSellQuantity })
     console.log('Offer_sell is: ', offer_sell, '\n\n')
     const actionOfferSellCreateParams = offer_sell.getActionParams()
     await contracts.daoreg.createoffer(...actionOfferSellCreateParams, { authorization: `${offer_sell.params.creator}@active` })
```

Comment:
`    await contracts.daoreg.createoffer(...actionOfferSellCreateParams, { authorization: `${offer_sell.params.creator}@active` })`

This is the only thing that should be on the // Act

Reply:
Done.